potodo/cache.py

```diff
@@ -33,6 +36,9 @@ def get_cache_file_content(
 def set_cache_content(
     obj: Dict[Path, PoFileStats], cache_args: Any, path: str = ".potodo/cache.pickle"
 ) -> None:
+    """
+    Set the cache.
```

Comment:
Sets the cache content to `obj` with args `cache_args` inside of the cache file `path`